app/settings/server/observer.js

```diff
@@ -1,19 +1,8 @@
-import { Meteor } from 'meteor/meteor';
-
-import { Settings } from '../../models/server';
 import { setValue } from './raw';
 
-const updateValue = (id, fields) => {
+export const updateValue = (id, fields) => {
```

Comment:
since the `observer` was removed from this file, this function could be moved as well and the file could be deleted..